app/models/Broadcast.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+/**
+ * Imports.
+ */
+const mongoose = require('mongoose');
+
+/**
+ * Schema.
+ */
+const broadcastSchema = new mongoose.Schema({
+
+  _id: { type: Number, index: true },
+  type: { type: String, enum: ['signup_prompt'] },
```

Comment:
Open to other names for this -- eventually there will be another type for `reportback_prompt` per #742